assets/sass/04-elements/links.scss

```diff
@@ -0,0 +1,24 @@
+/*
+ * text-underline-offset doesn't work in Chrome at all 👎
+ * But looks nice in Safari/Firefox, so let's keep it and
+ * maybe Chrome will support it soon.
+ */
+a {
+	color: var(--global--color-black);
+	text-underline-offset: 3px;
+	transition: all .15s ease-in-out;
+  }
+
+a:hover {
+	text-decoration-style: dotted;
+}
+
+/*
+ * The reason for the !important style here is the global focus styles created
+ * by the `.site *:focus {` rules in the 03-generic/reset.scss file.
+ */
+a:focus {
+	outline: none !important;
+	border: 1px solid var(--global--color-black);
```

Comment:
The problem with using a border here is that it takes up space, so when a link is focused there is a visible "jump" or shift of not only the link but the content that follows, because the extra pixels are added.

This makes a text difficult to read because the shifts are distracting.

![visible-shift-of-content](https://user-images.githubusercontent.com/7422055/94216269-4232a980-fedf-11ea-9d39-8f349ec44171.gif)

The shift can be avoided by adding a transparent border to the default state of the link, 
but that would create extra spacing and move the entire link so it is not a viable option.

Reply:
Ah, as per comment https://github.com/WordPress/twentytwentyone/issues/110#issuecomment-698423081 this would be changed back to an outline. :)